ee/vellum_ee/workflows/display/nodes/vellum/inline_prompt_node.py

```diff
@@ -87,7 +98,36 @@ def serialize(
             ],
             "ports": self.serialize_ports(display_context),
         }
-        attributes = self._serialize_attributes(display_context)
```

Comment:
we shouldn't need `blocks_contain_descriptors` during this serialize attributes method. We should only need it above when setting the `blocks: list` variable to empty or not